src/hooks/v2/NftRewards.ts

```diff
@@ -26,16 +28,22 @@ async function getRewardTierFromIPFS({
 
   const response = await axios.get(url)
   const ipfsRewardTier: IPFSNftRewardTier = response.data
+  const maxSupply = contractNftRewardTier.initialQuantity.eq(
+    BigNumber.from(DEFAULT_NFT_MAX_SUPPLY),
+  )
+    ? DEFAULT_NFT_MAX_SUPPLY
+    : contractNftRewardTier.initialQuantity.toNumber()
   return {
     name: ipfsRewardTier.name,
     description: ipfsRewardTier.description,
     externalLink: withHttps(ipfsRewardTier.externalLink),
-    contributionFloor: ipfsRewardTier.attributes.contributionFloor,
+    contributionFloor: parseFloat(
+      formatWad(contractNftRewardTier.contributionFloor) ?? '0',
+    ),
     tierRank: index + 1,
-    maxSupply: ipfsRewardTier.attributes.maxSupply ?? DEFAULT_NFT_MAX_SUPPLY,
+    maxSupply: maxSupply ?? DEFAULT_NFT_MAX_SUPPLY,
```

Comment:
**question** is `maxSupply` ever undefined anymore? Can we get rid of the fallback now?

Reply:
Yup